### DISCUSSION TITLE
Translations for content decoupled from system available-languages

### DISCUSSION THREAD

**Discussion by gastonbesada** [directus/directus#20660] [Feature Requests]
### Summary

Decouple from the data studio system available-languages to have a new API response locales setting for content translation locales for api responses.

### Motivation

Maybe I want to set the "Default Language" of the app to Spanish (es) translation but, this is not the "Default language" for the API. 

Basically the idea is to have the ability to create new locales used exclusively for translating all user content for the api responses (every value in every element of every collection), and ignore the set of available locales used for system translation in the Data studio app (the display view).

This feature maybe solve:
https://github.com/directus/directus/discussions/19761
https://github.com/directus/directus/discussions/3945
https://github.com/directus/directus/discussions/20523

### Detailed Design

- Add a new sub menu item in the Admin/Settings called "API locales".
- Add a system display with the list view and create button for the "API locales".
- Change the behavior of system-language to decouple system translations and API responses translations.

### Must Have:
- Create a new system "API locales" interface similar to: [this](https://docs.directus.io/guides/headless-cms/content-translations.html#create-a-languages-collection)
- Custom Translations, Collection, and Field names translations will remain as part of the system language interface.
- The new "API locales" setting will change the behavior of the existing system interface.
- Ability to select and change item content translations: If more than one locale has been configured in "API locales", a new dropdown with locales will be added next to the title (or sidebar) in the collection list display view and Item content edit view.
- Deleting "Translations" relational field, it would no longer make sense with this new approach.
- Add dedicated lang parameter: https://github.com/directus/directus/discussions/6654 and https://github.com/directus/directus/discussions/20523

### Should Have:
Fields for the new "API locales" setting form:
- Locale - pulldown - select one - populated with i18n available locales.
- Fallback locale - pulldown - select - the main locale for delivery content if the translations for this locale are incomplete.
- Enable in response - check box
- Enable in editing - check box
- Allow empty fields - check box

### Could Have:
- For adding a new locale, the user will be able to select only the available i18n language code options:
https://github.com/ladjs/i18n-locales
https://github.com/ladjs/country-language

### Drawbacks

.

### Alternatives

.

### Adoption Strategy

.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
